Tensile/SolutionStructs.py

```diff
@@ -3077,7 +3077,13 @@ def assignDerivedParameters(state):
     state["LocalWriteUseSgprA"] = False
     state["LocalWriteUseSgprB"] = False
 
-    state["WorkGroupMapping" ] = abs(state["WorkGroupMapping"])
+    if state["KernelLanguage"] != "Assembly" or state["WorkGroupMappingType"] == "Z":
+      # WGM < 0 is only for Asm + WorkGroupMappingType==B
+      # convert to > 0
+      state["WorkGroupMapping" ] = abs(state["WorkGroupMapping"])
+    if state["WorkGroupMapping" ] == -1:
+      # -1 code is same as 1. Convert to 1.
+      state["WorkGroupMapping" ] = 1
```

Comment:
Do asm kernels support WGMType=='z'? I only found code in KernelWriterAssembly related to 'B' mode